content/intro-to-storybook/angular/en/test.md

```diff
@@ -65,12 +65,11 @@ yarn add storybook-chromatic
 Import Chromatic in your `.storybook/config.js` file.
 
 ```javascript
-import { configure } from "@storybook/angular";
-import "storybook-chromatic";
-import "../src/styles.less";
+import { configure } from '@storybook/angular';
+import '../src/styles.less';
 
 // automatically import all files ending in *.stories.ts
-const req = require.context("../src/", true, /.stories.ts$/);
+const req = require.context('../src/', true, /.stories.ts$/);
```

Comment:
This is kinda an annoyance. I have manually turned off formatting on save for these MD files before when editing them, but I think this should be addressed eventually:

https://github.com/chromaui/learnstorybook.com/issues/162